docker-multinode/common.sh

```diff
@@ -0,0 +1,437 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Utility functions for Kubernetes in docker setup
+# Authors @luxas @wizard_cxy @resouer @loopingz 
+
+# Variables
+K8S_VERSION=${K8S_VERSION:-"1.2.0-alpha.8"}
+ETCD_VERSION=${ETCD_VERSION:-"2.2.1"}
+FLANNEL_VERSION=${FLANNEL_VERSION:-"0.5.5"}
+FLANNEL_IFACE=${FLANNEL_IFACE:-"eth0"}
```

Comment:
The name eth0 is deprecated. New distros generate other names like 'enp0s25'

I think the following would fix:
DEFAULT_IFACE=$(ip -o -4 route show to default | awk '{print $5}')
FLANNEL_IFACE=${FLANNEL_IFACE:-$DEFAULT_IFACE}

(grabs the interface with the default route)